ibm_catalog.json

```diff
@@ -1585,6 +1666,20 @@
                 }
               ]
             },
+            {
+              "key": "resource_group",
```

Comment:
same as above

Reply:
I can update these but they are virtual inputs that would get a display name of `resource_group` so I skipped calling them existing